code/game/objects/items/plushes.dm

```diff
@@ -662,18 +662,13 @@
 	attack_verb_continuous = list("flutters", "flaps")
 	attack_verb_simple = list("flutter", "flap")
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
-///Used to track how many people killed themselves with item/toy/plush/moth
-	var/suicide_count = 0
+
+/obj/item/toy/plush/moth/Initialize(mapload)
+	AddComponent(/datum/component/suicide_count, SUICIDE_VIS_NONE, \
```

Comment:
This doesn't need to have a newline extender, this isn't a define.

Reply:
yes it is

Reply:
AddComponent is a sekrit macro for `_AddComponent(list(...))` so that it supports named parameters